site/_resources/how-we-work.md

```diff
@@ -25,16 +33,52 @@ Everything flows from this statement.
 - [Talk about what you intend to do, then do the thing you talked about.][1]
 GitHub review tools suck for extended debate, if you find you’re taking past your reviewer, its a sign that more design is needed.
 
-- Log an issue or it didn’t happen. 
+## Coding Practices
 
 - Before you fix a bug, write a test to show you fixed it.
 
 - Before you add a feature, write a test so someone else doesn’t break your feature by accident.
 
-- When submitting a PR add the appropriate release milestone and also add the Github label "release note" if this PR warrants getting called out in the next release.
-
 - You are permitted to refactor as much as you like to achieve these goals.
 As Kent beck said, ["make the change easy, then make the easy change."][2]
 
+## Github Labels
+
+Github issues should be triaged and have their status recorded.
+Triaging issues and labeling them appropriately makes it easy for the issue submitter and other contributors to see what the state of an issue is at any time.
+The goal of triaging an issue is to make a decision about what should be done with it.
+The issue should investigated enough to fully understand and document the problem and then decide whether the issue should be addressed by the project.
+It may not be necessary to decide how an issue should be addressed during triage, since that could involve substantial research and design.
+
+- To be considered triaged, an issue must have the "lifecycle/accepted" label.
```

Comment:
Maybe something like `lifecycle/triaged`? "accepted" sounds like the ideas in the issue are ready to be implemented by a contributor. We should be clear about the meaning of "accepted" if that's what's used in this example.

Reply:
So "accepted" is meant to mean that a maintain agrees that this issue is a real thing. That it is a bug or a feature that is in scope for Contour.

We could directly adopt the Kubernetes labels (e.g. "triage/accepted"  Issue has been triaged by a SIG representative and is ready to be worked)

https://github.com/kubernetes/enhancements/tree/master/keps/sig-contributor-experience/1553-issue-triage